spanner/tests/system/test_system.py

```diff
@@ -282,6 +283,18 @@ def test_create_database(self):
             for database in Config.INSTANCE.list_databases()]
         self.assertIn(temp_db_id, database_ids)
 
+    def test_create_database_not_found(self):
+        temp_db_id = 'temp_db' + unique_resource_id('_')
+        temp_db = Config.INSTANCE.database("bitemporal_pattern1", ddl_statements=[
+            """CREATE TABLE MyTable (
+                Id        String(36) NOT NULL,
+                Field1  STRING(36) NOT NULL
+            ) PRIMARY KEY (Id)""",
+            """CREATE INDEX IDX ON yTable (Field1)"""
+        ])
+        with self.assertRaisesRegexp(NotFound, "Database not found"):
```

Comment:
1. Please use single quotes
1. Any reason not to just check the entire message?

Reply:
My reasoning was that "Database not found" was the crucial part.  The message might fluctuate a little?